Help compiling example

Hello,

I am trying to run the example: "examples/simple.c" but I am having compiler errors.

I am new to using C libraries with `gcc` so I am probably missing something obvious.

Here are the commands I ran.

```sh
(base) nathan@p3070:~/github$ git clone git@github.com:Blosc/c-blosc.git
Cloning into 'c-blosc'...
remote: Enumerating objects: 9333, done.
remote: Counting objects: 100% (1586/1586), done.
remote: Compressing objects: 100% (813/813), done.
remote: Total 9333 (delta 900), reused 1407 (delta 759), pack-reused 7747
Receiving objects: 100% (9333/9333), 8.44 MiB | 40.58 MiB/s, done.
Resolving deltas: 100% (6117/6117), done.

(base) nathan@p3070:~/github$ cd c-blosc/examples/

(base) nathan@p3070:~/github/c-blosc/examples$ gcc -O3 -msse2 simple.c -I../blosc -o simple -L../build/blosc
/usr/bin/ld: /tmp/ccyHg8k8.o: in function `main':
simple.c:(.text.startup+0x6a): undefined reference to `blosc_init'
/usr/bin/ld: simple.c:(.text.startup+0x9a): undefined reference to `blosc_compress'
/usr/bin/ld: simple.c:(.text.startup+0xfc): undefined reference to `blosc_decompress'
/usr/bin/ld: simple.c:(.text.startup+0x114): undefined reference to `blosc_destroy'
collect2: error: ld returned 1 exit status
```

I am trying to run the command in https://github.com/Blosc/c-blosc/blob/4f8541c8885bcba7d141ac1a270a9b5a6afe97d3/examples/simple.c#L12-L14

Is there a step I missed?

--- a/examples/noinit.c
+++ b/examples/noinit.c
@@ -12,8 +12,7 @@
 
     or, if you don't have the blosc library installed yet:
 
-    $ gcc -O3 -msse2 noinit.c -I../blosc -o noinit -L../build/blosc
-    $ export LD_LIBRARY_PATH=../build/blosc
+    $ gcc -O3 -msse2 noinit.c ../blosc/*.c  -I../blosc -o noinit
 
     Using MSVC on Windows:
 

--- a/examples/noinit.c
+++ b/examples/noinit.c
@@ -12,7 +12,7 @@
 
     or, if you don't have the blosc library installed yet:
 
-    $ gcc -O3 -msse2 noinit.c ../blosc/*.c  -I../blosc -o noinit
+    $ gcc noinit.c -I../blosc -o noinit ../build/blosc/libblosc.a
 
     Using MSVC on Windows:
 

--- a/examples/simple.c
+++ b/examples/simple.c
@@ -11,7 +11,7 @@
 
     or, if you don't have the blosc library installed:
 
-    $ gcc -O3 -msse2 simple.c -I../blosc -o simple -L../build/blosc
+    $ gcc -O3 -msse2 simple.c ../blosc/*.c  -I../blosc -o simple
 
     Using MSVC on Windows:
 

--- a/examples/simple.c
+++ b/examples/simple.c
@@ -11,7 +11,7 @@
 
     or, if you don't have the blosc library installed:
 
-    $ gcc -O3 -msse2 simple.c ../blosc/*.c  -I../blosc -o simple
+    $ gcc simple.c -I../blosc -o simple ../build/blosc/libblosc.a
 
     Using MSVC on Windows:
 

